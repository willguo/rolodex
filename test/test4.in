J'aime Devoi, gray, 38410, 628 102 3672
Mary-Lee B. S. D-vorak, black, 23222, 121 234 1231
J'aime, gray, 38410, 628 102 3672
MaryAnn B D-vorak, black, 23222, 121 234 1231
Mary-Ann B D-vorak, black, 23222, 121 234 1231
arthur z d, random, 12302, 293 294 2048
d, arthur, (666)-666-6666, brown, 12312
d, arthur, (666)-666-6666, brown, 12312
d, arthur, (666)-666-6666, brackish, 12342
d, arthur, (666)-666-6662, brown, 12311