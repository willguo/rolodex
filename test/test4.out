{
  "entries": [
    {
      "color": "black", 
      "firstname": "Mary-Ann B", 
      "lastname": "D-vorak", 
      "phonenumber": "121-234-1231", 
      "zipcode": "23222"
    }, 
    {
      "color": "black", 
      "firstname": "Mary-Lee B. S.", 
      "lastname": "D-vorak", 
      "phonenumber": "121-234-1231", 
      "zipcode": "23222"
    }, 
    {
      "color": "black", 
      "firstname": "MaryAnn B", 
      "lastname": "D-vorak", 
      "phonenumber": "121-234-1231", 
      "zipcode": "23222"
    }, 
    {
      "color": "gray", 
      "firstname": "J'aime", 
      "lastname": "Devoi", 
      "phonenumber": "628-102-3672", 
      "zipcode": "38410"
    }, 
    {
      "color": "brown", 
      "firstname": "arthur", 
      "lastname": "d", 
      "phonenumber": "666-666-6662", 
      "zipcode": "12311"
    }, 
    {
      "color": "brown", 
      "firstname": "arthur", 
      "lastname": "d", 
      "phonenumber": "666-666-6666", 
      "zipcode": "12312"
    }, 
    {
      "color": "random", 
      "firstname": "arthur z", 
      "lastname": "d", 
      "phonenumber": "293-294-2048", 
      "zipcode": "12302"
    }
  ], 
  "errors": [
    2, 
    7, 
    8
  ]
}